CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 2.13.0 (2018-10-23)
+- Updated the [skyux-builder-plugin-stache](https://github.com/blackbaud/skyux-builder-plugin-stache/blob/master/CHANGELOG.md#151-2018-08-02) to `1.7.0`.
+  - This update includes the new `skyux stache-update` command to update their SKY UX, SKY UX Builder, and Stache dependencies for them, deleting node_modules and npm installing.
```

Comment:
What do you think of changing this to "Adds the `skyux stache-update` command to update SKY UX, SKY UX Builder, and Stache dependencies. The command also deletes the `node_modules` folder and runs `npm install`."?

Reply:
Much better.